templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -34,6 +38,8 @@ Metadata:
         default: ASI identifier
       KeyPairName:
         default: Key Name
```

Comment:
In the Jira, Confluence, Bitbucket, and Crowd templates, the `KeyPairName` default name is set to 

> SSH Key Pair Name

Can we change it here for consistency?